moveit_core/constraint_samplers/include/moveit/constraint_samplers/default_constraint_samplers.h

```diff
@@ -521,6 +521,8 @@ class IKConstraintSampler : public ConstraintSampler
   std::string ik_frame_;                                          /**< \brief Holds the base from of the IK solver */
   bool transform_ik_; /**< \brief True if the frame associated with the kinematic model is different than the base frame
                          of the IK solver */
+  bool eef_to_ik_tip_;
```

Comment:
please add a comment here that explains the semantics of the flag just like the one for `transform_ik_` above.